service/ArbitrageService.js

```diff
@@ -42,23 +50,33 @@ class ArbitrageService {
 			arbitrageBlxmBalance = await this._ethContracts.blxmTokenContract.getTokenBalance(constants.ARBITRAGE_WALLET_ADDRESS);
 			adjustmentValue = AdjustmentValueService.getAdjustmentValue(balanceBlxmETH, balanceUsdcETH, balanceBlxmBSC, balanceUsdcBSC);
 
+			logger.info("The BLXM token trades cheaper on the Binance Smart Chain network than Ethereum network" + '\n' + 
+						"Price difference: " + ethers.utils.formatEther((poolPriceEth-poolPriceBsc)) + " USD");
+
 			result = await this.startArbitrageTransferFromBSCToETH(adjustmentValue, arbitrageBlxmBalance, poolPriceBsc, poolPriceEth, balanceUsdcETH);
 		    profit = this._calculateAbitrageProfit(result.swapAmount, balanceBlxmETH, balanceBlxmBSC, result.profit, "ETH");
 		} 
 		else {
 			arbitrageBlxmBalance = await this._bscContracts.blxmTokenContract.getTokenBalance(constants.ARBITRAGE_WALLET_ADDRESS);
 			adjustmentValue = AdjustmentValueService.getAdjustmentValue(balanceBlxmBSC, balanceUsdcBSC, balanceBlxmETH, balanceUsdcETH);
 
+			logger.info("The BLXM token trades cheaper on the Ethereum network than Binance Smart Chain network" + '\n' + 
+						"Price difference: " + ethers.utils.formatEther((poolPriceBsc-poolPriceEth)) + " USD");
+
 			result = await this.startArbitrageTransferFromETHToBSC(adjustmentValue, arbitrageBlxmBalance, poolPriceBsc, poolPriceEth, balanceUsdcETH);
 			profit = this._calculateAbitrageProfit(result.swapAmount, balanceBlxmBSC, balanceBlxmETH, result.profit, "BSC");
 		}
 
-		this._databaseService.AddData({Profit: profit});
+		//this._databaseService.AddData({Profit: profit});
 	}
 
 	async startArbitrageTransferFromBSCToETH(amount, arbitrageBlxmBalance, poolPriceBsc, poolPriceEth, balanceUsdc) {
+		
 		// is liqudity available ? 
 		if (!arbitrageBlxmBalance.isZero()) {
```

Comment:
Siehe anderen Kommentar zur Liquidität in function startArbitrageTransferFromETHToBSC